avm/res/app/managed-environment/tests/e2e/defaults/dependencies.bicep

```diff
@@ -18,5 +23,37 @@ resource logAnalyticsWorkspace 'Microsoft.OperationalInsights/workspaces@2022-10
   })
 }
 
+resource virtualNetwork 'Microsoft.Network/virtualNetworks@2023-04-01' = {
```

Comment:
Can the virtual network resource ID be referenced instead of being created at resource module level?